apps/beets-frontend-v3/app/layout.tsx

```diff
@@ -23,11 +21,9 @@ export default function RootLayout({ children }: PropsWithChildren) {
         style={{ marginRight: '0px !important' }} // Required to prevent layout shift introduced by Rainbowkit
         suppressHydrationWarning
       >
-        <ColorThemeProvider defaultTheme={DEFAULT_THEME_COLOR_MODE}>
-          <ThemeProvider>
-            <Providers>{children}</Providers>
-          </ThemeProvider>
-        </ColorThemeProvider>
+        <ThemeProvider>
```

Comment:
Why remove ColorThemeProvider here?

Reply:
Groninge removed it from beets to demonstrate that the hydration issue is due to this (beets only has dark mode)

Reply:
it looks like next-themes is no longer needed but i'll admit i only looked briefly
as this only concerns balancer it might need more testing

Reply:
@agualis can you look into theming with chakra/nextjs. It used to be that using next-themes was the default approach. I think there might have also been a reason to use it so that the theme applied correctly in RSCs.